assets/javascripts/kitten/components/buttons/button.js

```diff
@@ -22,8 +22,13 @@ export class Button extends React.Component {
 
     const Tag = tag
 
+    // Adds keyboard accessibility to `<a>`
+    const tabindex = this.props.tag == "a" && !this.props.href ? 0 : null
```

Comment:
Je ne comprend pas bien cette partie du code : `!this.props.href ? 0 : null`, ça te renverra toujours `true`, non ?
- `!0` => `true`
- `!null` => `true`